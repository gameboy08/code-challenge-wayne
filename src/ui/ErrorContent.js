import React from "react";
import "./ErrorContent.scss";
export default function ErrorContent(props) {
    const {children} = props;
  return (
    <div className="error-container">
      {children}, please go back to{" "}
      <a href="http://localhost:3000/ ">http://localhost:3000/ </a>
    </div>
  );
}
