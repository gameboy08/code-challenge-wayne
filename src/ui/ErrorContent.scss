@import "../utils/variables";
.error-container {
    height: 100vh;
    display: flex;
    justify-content: center;
    align-items: center;
    background: $light;
    font-size: 2.0rem;
    font-weight: bold;
}