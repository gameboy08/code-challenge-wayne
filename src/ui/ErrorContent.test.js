import React from "react";
import renderer from "react-test-renderer";
import ErrorContent from "./ErrorContent";

it("ErrorContent renders correctly", () => {
  const tree = renderer.create(<ErrorContent>Network error</ErrorContent>).toJSON();
  expect(tree).toMatchSnapshot();
});
