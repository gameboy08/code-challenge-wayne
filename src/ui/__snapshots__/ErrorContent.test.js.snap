// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorContent renders correctly 1`] = `
<div
  className="error-container"
>
  Network error
  , please go back to
   
  <a
    href="http://localhost:3000/ "
  >
    http://localhost:3000/ 
  </a>
</div>
`;
